City-of-Helsinki/hel-fi-drupal-grants
# test: AU-1244: Add initial tests for liikuntatapahtuma form

# [AU-1244](https://helsinkisolutionoffice.atlassian.net/browse/AU-1244)


Improve development by catching issues early.

Currently there is webform related issue in budget components module. That needs resolving. This also need more fields under testing. 

## What was done


* Add tests related for liikunta tapahtuma form.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/AU-1244-liikunta-tapahtuma`

## How to test


  * Run tests. Check command in README 



[AU-1244]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1244?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
